prototype2/common/src/world.rs.in

```diff
@@ -69,7 +68,24 @@ pub struct ClientWorld {
   pub disabled: HashSet<Entity>
 }
 
-pub type Entity = Uuid;
+pub struct ClientWorld2 {
+  #[allow(dead_code)]
```

Comment:
Ah yeah. I need to delete this for now.


The story is that there are basically two ECSes: one for the server and one for the client. In this PR I updated the server but not the client. This is an old struct from when I was going to do both upfront.

On warning supression: I don't think rustc is good enough, but i think Clippy is.